tests/rptest/tests/acls_test.py

```diff
@@ -49,16 +50,34 @@ def setUp(self):
         # it with custom security settings
         return
 
+    def authz_enabled(self, use_sasl, enable_authz):
+        if enable_authz != None and enable_authz:
```

Comment:
Nit: in general prefer `is None` and `is not None` as equality operators can be overloaded.